features/step_definitions/cluster_version.rb

```diff
@@ -0,0 +1,10 @@
+
+Given(/^the "([^"]*)" capability is enabled$/) do |capability|
```

Comment:
@dpuniaredhat I've added the step definition here.  You can now remove the step definition in your PR and just use this one instead.

Reply:
Thanks